plutus-core-spec/plutus-core-specification.tex

```diff
@@ -93,7 +95,18 @@
 \newcommand{\builtin}[3]{\construct{\keyword{builtin} ~ #1 ~ #2 ~ #3}}
 \newcommand{\error}[1]{\construct{\keyword{error} ~ #1}}
 
+%% Extra untyped terms
+\newcommand{\lamU}[2]{\construct{\keyword{lam} ~ #1 ~ #2        }}
+\newcommand{\appU}[2]{\texttt{[} #1 ~ #2 \texttt{]}}
+\newcommand{\errorU}{\construct{\keyword{error}}}
+\newcommand{\builtinU}[2]{\construct{\keyword{builtin} ~ #1 ~ #2}}
+\newcommand{\delay}[1]{\construct{\keyword{delay} ~ #1}}
+\newcommand{\force}[1]{\construct{\keyword{force} ~ #1}}
 
+\newcommand{\erase}[1]{\llbracket#1\rrbracket}
+%\newcommand{\erase}[1]{\left|#1\right|}
+%\newcommand{\erase}[1]{\overline{#1}}
+%\newcommand{\erase}[1]{\left\lVert#1\right\rVert}
```

Comment:
Pick one and keep it!

Reply:
Thanks.  I stuck these in when I was experimenting to see which one looked best and then forgot about them.